src/file.cpp

```diff
@@ -62,6 +62,7 @@ POSSIBILITY OF SUCH DAMAGE.
 
 #ifdef TORRENT_DISK_STATS
 #include "libtorrent/io.hpp"
+#include "libtorrent/storage.hpp"
```

Comment:
if you remove the line you added to the top of silly_hash(), everything would still work, right? That's just an optimization. I'm suggesting that the optimization may not be worth the additional dependency. Also, if the disk access log is disabled, will silly_hash() ever be called?